<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Suchi" name="parallel.ParallelRun" tests="5" failures="1" timestamp="2022-05-15T23:52:36 IST" time="67.052" errors="0">
  <testcase name="runScenario" time="15.537" classname="parallel.ParallelRun"/>
  <system-out/>
  <testcase name="runScenario" time="12.645" classname="parallel.ParallelRun">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at parallel.Homepagestepdefinition.user_gets_home_section(Homepagestepdefinition.java:51)
at âœ½.user gets Home section(classpath:parallel/Homepage.feature:11)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="11.633" classname="parallel.ParallelRun"/>
  <system-out/>
  <testcase name="runScenario" time="13.618" classname="parallel.ParallelRun"/>
  <system-out/>
  <testcase name="runScenario" time="13.619" classname="parallel.ParallelRun"/>
  <system-out/>
</testsuite> <!-- parallel.ParallelRun -->
